aiida/cmdline/commands/cmd_computer.py

```diff
@@ -216,13 +218,15 @@ def _computer_use_login_shell_performance(transport, scheduler, authinfo):  # py
 
     echo.echo_debug(f'Execution time: {timing_true} vs {timing_false} for login shell and normal, respectively')
 
-    if timing_true - timing_false > tolerance:
-        message = (
-            'computer is configured to use a login shell, which is slower compared to a normal shell (Command execution'
-            f' time of {timing_true} s versus {timing_false}, respectively).\nUnless this setting is really necessary, '
-            'consider disabling it with: verdi computer configure core.local COMPUTER_NAME -n --no-use-login-shell'
+    if not isclose(timing_true, timing_false, rel_tol=rel_tol, abs_tol=abs_tol):
+        return True, (
+            '\nWarning: the computer is configured to use a login shell, which is slower compared to a normal shell.\n'
```

Comment:
Just a tiny nitpick: I think an extra end of line here would make the message clearer.

```suggestion
            '\n\nWarning: the computer is configured to use a login shell, which is slower compared to a normal shell.\n'
```

Ideally the `Warning:` would also be in orange, but if that is not possible I would also settle for a ❗️emoji 😇.